packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -194,17 +194,40 @@ class Tooltip extends StatefulWidget {
   ///  * [Feedback], for providing platform-specific feedback to certain actions.
   final bool? enableFeedback;
 
-  static final Set<_TooltipState> _openedToolTips = <_TooltipState>{};
+  static final List<_TooltipState> _openedTooltips = <_TooltipState>[];
+
+  // Causes any current tooltips to be "covered". Only called for mouse hover enter
```

Comment:
I was a little confused when I first saw the term "covered' here. This really isn't covering the other tooltips so much as hiding them. Perhaps "hidden" would be a better term here? Or maybe call the current tooltip the active one and the others are inactive?

Reply:
Well, I do agree about the name, but "_hideTooltip" is already a thing that means something entirely different.

Hmm.  Maybe I'll rename `_hideTooltip` to `_dismissTooltip` and switch from "cover" to hide that way.

Reply:
Actually, I tried to do this, and the problem is that the opposite of "hide" is "show", and so changing the name of "_hideTooltip" would mean changing "_showTooltip" too, and also the public API value "showDuration", which would be breaking.

How about "conceal" and "reveal" instead of "cover" and "uncover"?

Reply:
Bummer. Yeah, "conceal" would be better than "cover" to me. How about the notion of having a tooltip be "active" or not? There would only ever be one active tooltip, correct?